components/Tile/TileSet.tsx

```diff
@@ -1,17 +1,44 @@
+import styled from "styled-components";
+import css from "@styled-system/css";
 import Flex from "components/Flex";
 import Tile from "./Tile";
 import TileList from "./TileList";
 
 type TileType = React.ReactElement<typeof Tile>;
 type TileListTyle = React.ReactElement<typeof TileList>;
 
+export interface TileWrapperProps {
+  children: React.ReactNode;
+}
+
+export const TileWrapper = ({ children }: TileWrapperProps) => {
+  return <StyledWrapper>{children}</StyledWrapper>;
+};
+
+export const StyledWrapper = styled(Flex)(
+  css({
+    flex: "0 0 100%",
+    justifyContent: "stretch",
+    alignItems: "stretch",
+    maxWidth: ["100%", "calc((100% - 64px) / 3)"],
+    pb: [3, 5],
+  })
+);
 export interface TileSetProps {
   children: TileType | TileListTyle | Array<TileType | TileListTyle>;
 }
 
 const TileSet = ({ children }: TileSetProps) => {
   return (
-    <Flex flexWrap="wrap" alignItems="stretch">
+    <Flex
+      flexWrap="wrap"
+      alignItems="stretch"
+      css={css({
+        [`& ${StyledWrapper}:nth-child(3n+2), & ${StyledWrapper}:nth-child(3n)`]: {
```

Comment:
We can also consider this one selector:
```suggestion
        [`& ${StyledWrapper}:not(:nth-child(3n+1))`]: {
```
it looks a little bit simpler to me

Reply:
We need to also change default margin and margin value inside this selector to make it work.

Right now `3n+1` will select 1, 4, 7, etc elements. But original selector was selecting all elements except these ones.

Will change it in original code.